prospector/commit_processor/feature_extractor.py

```diff
@@ -109,6 +115,18 @@ def is_commit_in_given_interval(
         )
 
 
+def extract_referred_to_by_nvd(
+    commit: Commit, advisory_record: AdvisoryRecord, nvd_rest_endpoint: str
+) -> bool:
+    response = requests.get(
+        nvd_rest_endpoint + "/nvd/vulnerabilities/" + advisory_record.vulnerability_id
+    ).json()
+    references = response["cve"]["references"]["reference_data"]
```

Comment:
Also, note taht the JSON returned by our own endpoint is not exactly the same as the one returned from the NVD webservice: the latter, wraps data within a larger data structure, whereas our server only returns the inner data (which is what matters for us).
Bottom line: to make the two services interchangeable would require some tweaks...

Reply:
Should the function try to figure out which kind of json it got and then extract the required data?

Reply:
Well, if we want the two to be interchangeable, we have to do that; not urgent, and I am not sure what is the cleanest way to do that. Your thoughts on this?